src/test/java/org/javarosa/xform/parse/ExternalSecondaryInstanceParseTest.java

```diff
@@ -184,14 +183,14 @@ public void realInstanceIsResolved_whenFormIsDeserialized_afterPlaceholderInstan
 
     @Test
     // Clients would typically catch this exception and try parsing the form again which would succeed by using the placeholder.
-    public void fileNotFoundException_whenFormIsDeserialized_afterPlaceholderInstanceUsed_andFileStillMissing() throws IOException, DeserializationException {
+    public void ioException_whenFormIsDeserialized_afterPlaceholderInstanceUsed_andFileStillMissing() throws DeserializationException {
```

Comment:
I might just be being slow today, but I'm missing the reason for this change?

Reply:
Good catch! In versions 1.6+ of the lib some other `IOException` exception is thrown. It's not in Android 21 so that's why I had to downgrade the library further. Forgot to clean this up. I rebased -- hopefully you won't find it too hard to follow.